artacle-swagger.yaml

```diff
@@ -825,6 +827,641 @@ paths:
+      parameters:
+        - name: cid
+          in: path
+          required: true
+          description: Artacle collection ID as returned by the /collections/ endpoint
+          schema:
+            type: integer
+            example: 78
+        - name: apiKey
+          in: query
+          required: true
+          description: API access key
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - data
+                properties:
+                  data:
+                    type: array
+                    description: Array of tokens data
+                    items:
+                      type: object
+                      required: 
+                        - tokenId
+                        - tokenName
+                        - currentOwner
+                        - AValuation
+                        - AValuationUSD
+                        - topTraitValuation
+                        - topTraitValuationUSD
+                        - rarityReward
+                        - rarityRewardUSD
+                        - isSpecialCase
+                        - ARank
+                        - rarityRank
+                        - valuationTimestamp
+                        - mintPrice
+                        - mintPriceUSD
+                        - listingPrice
+                        - listingPriceUSD
+                        - listingTimestamp
+                        - isListed
+                        - lastSalePrice
+                        - lastSalePriceUSD
+                      properties:
+                        tokenId:
+                          type: string
+                          description: Token ID as per smart contract (for example, 1 for Bored Apes Yacht Club or 78000001 for Art Blocks Fidenza)
+                          example: 78000001
+                        tokenName:
+                          type: string
+                          description: Token name
+                          example: Floris Veilstorm
+                        currentOwner:
+                          type: string
+                          description: Blockchain address of current token owner
+                          minLength: 42
+                          example: "0x9df947cd8a9810964473339c38a95201ecbe112d"
+                        AValuation:
+                          type: number
+                          description: Token A-Valuation
+                          format: float
+                          example: 0.0414
+                        AValuationUSD:
+                          type: number
+                          description: Token A-Valuation in USD
+                          format: float
+                          example: 102.768876
+                        topTraitValuation:
```

Comment:
Here and after - see my previous comments about the description of these fields.